src/js/rise-cache.js

```diff
@@ -58,26 +58,20 @@ RiseVision.Common.RiseCache = (function () {
     r.send();
   }
 
-  function getFile(fileUrl, callback, nocachebuster) {
+  function getFile(fileUrl, callback) {
     if (!fileUrl || !callback || typeof callback !== "function") {
       return;
     }
 
     function fileRequest() {
-      var url, str, separator;
+      var url = "";
 
-      if (_isCacheRunning) {
-        // configure url with cachebuster or not
-        url = (nocachebuster) ? BASE_CACHE_URL + "?url=" + encodeURIComponent(fileUrl) :
-        BASE_CACHE_URL + "cb=" + new Date().getTime() + "?url=" + encodeURIComponent(fileUrl);
+      if (_isV2Running) {
```

Comment:
@donnapep I think this conditional statement should start with `if (_isCacheRunning)` and if it is true, inside that block determine `url` based on if `isV2Running` is true.